### ISSUE TITLE
Formatting inserts a space above header comments that breaks pico-8 cart label

### ISSUE THREAD

**Issue by schoblaska** [japhib/pico8-ls#51]
When the top of my file looks like this:

```
pico-8 cartridge // http://www.pico-8.com
version 41
__lua__
-- sokotiles
-- by schoblaska
```

The cart label gets generated like this:

<img width="699" alt="image" src="https://github.com/japhib/pico8-ls/assets/29129/8432d777-c3bc-492b-b7ff-f6e4bd09b880">

But if I have `pico8-ls` format the file, it inserts an empty line like this:

```
pico-8 cartridge // http://www.pico-8.com
version 41
__lua__

-- sokotiles
-- by schoblaska
```

and now the cart label only includes the first comment:

<img width="723" alt="image" src="https://github.com/japhib/pico8-ls/assets/29129/8a85205f-343e-404d-b525-d09f933c7fcd">

**Comment by japhib** [comment]
Does the formatter insert the space at the top of the file?

**Comment by schoblaska** [comment]
I believe so, yes. In my editor, the empty line is only inserted when the pic8-ls extension is enabled.

**Comment by akd-io** [comment]
Faced the same issue. Simple workaround is just removing it, running `> File: Save without Formatting` before exporting the `.p8.png`. But would be nice with a patch.

**Comment by akd-io** [comment]
Added #65 to try to resolve the issue

**Comment by japhib** [comment]
Released with 0.5.6 just now
